# Generated for NaoQi method ALBehaviorManager.addDefaultBehavior
# Set the given behavior as default

# The behavior name
string name
---


